Visits/Modules/Sources/APIEnvironmentLive/Trips.swift

```diff
@@ -75,6 +75,15 @@ extension Order: Decodable {
     let values = try decoder.container(keyedBy: CodingKeys.self)
 
     let id = try values.decode(ID.self, forKey: .id)
+
+    guard id.string.rangeOfCharacter(from: .whitespacesAndNewlines) == nil else {
```

Comment:
Haven't we agree that IDs can only contain alphnumericals and _ - ?

Reply:
@elagrion they agreed to make this change on frontend, but they can easily adjust this to be something else, while Visits app users won't update the app as quickly enough. That's why I want to add minimal constraints here and only look for what produces correct URLs and what doesn't. This way the app can still stay flexible enough for consuming data.

Data creation on the app is another story, I think for creating orders we should enforce the same that platform/frontend is doing.